server/lib/allowed-features.ts

```
@@ -7,6 +7,7 @@ import PlatformConstants from '../constants/platform';
 import { Forbidden } from '../graphql/errors';
 import { Loaders } from '../graphql/loaders';
 import { Collective, PlatformSubscription } from '../models';
+import { PROJECTS_ALLOWED_ACCOUNT_TYPES } from './collectivelib';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)